### ISSUE TITLE
What are the meannings of avaiable keys?

### ISSUE THREAD

**Issue by c02y** [peco/peco#466]
# What is the peco version, OS, architecture? 
                                                                                                                                                                                                   
peco version v0.5.3 (built with go1.10)
Ubuntu 16.04 LTS 64-bit

I see you list a lot of available keys at https://github.com/peco/peco#available-keys but you never mentioned what the keys are used for, such as how to `Select Multiple Lines`, I figure it out `C-SPC` to make the select line by accident.\

Why not add column in the `Available keys` table for usage of every keybinding.

**Comment by lestrrat** [comment]
What you wanted was the "Keymap" https://github.com/peco/peco#default-keymap

Keys are what can be assigned to an action, which are listed here https://github.com/peco/peco#available-actions
The default keymap shows which key or key sequence invokes the action.

**Comment by lestrrat** [comment]
Also, an individual key, such as the `Ctrl-A` does not *necessarily* correspond to an action, because we allow key sequences to be bound to actions https://github.com/peco/peco#key-sequences

We currently may not have sequences in the default keymap, but who knows, in the future we might, and then it would be confusing to so have an extra column in https://github.com/peco/peco#available-keys.

**Comment by c02y** [comment]
Sorry about the complains, but as a new user, I really don't know how to use the keybindings shown in gif demos.

Maybe you should tell more such as keybindings when showing the gifs.

**Comment by lestrrat** [comment]
I can sympathize with wanting to tweak the docs, but It's hard to follow what exactly you want. It would be great if you can create a PR that does the changes you want, and then maybe we can discuss specifics.
